src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -121,6 +147,30 @@ public void play(IClient client, RequestFunction function, AMFDataList params)
         }
 	}
 
+    /**
+     * Initalizes the Honeybadger error reporting tool. This is a public method so we can call
+     * it from the tests. It's outside the constructor, since testing constructors with Mockito is a pain.
+     */
+    public void initHoneybadger()
+    {
+        if(environment == null)
```

Comment:
hahahah you have a thing about spaces after if statements.  But I don't really care.